test/drivers/test_suites/syscall_exit_suite/pidfd_getfd_x.cpp

```diff
@@ -0,0 +1,60 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_pidfd_getfd
+
+TEST(SyscallExit, pidfd_getfdX)
+{
+    auto evt_test = get_syscall_event_test(__NR_pidfd_getfd, EXIT_EVENT);
+
+    evt_test->enable_capture();
+    
+
+    /*=============================== TRIGGER SYSCALL ===========================*/
+
+    int pid_fd = 0;
+    int target_fd = 0;
```

Comment:
I would use random vaules to test if our fillers correctly work
```suggestion
    int pid_fd = -1;
    int target_fd = 1023;
```